src/setup/testnet/mod.rs

```diff
@@ -11,23 +11,29 @@ use std::{
 use crate::{
     setup::{
         build_ripple_work_path,
-        node::{Node, NodeBuilder},
+        node::{Node, NodeBuilder, NodeType},
     },
-    tools::constants::{DEFAULT_PORT, TESTNET_NETWORK_ID},
+    tools::constants::{DEFAULT_PORT, TESTNET_NETWORK_ID, VALIDATORS_FILE_NAME},
 };
 
 /// Testnet's directory for nodes' configs.
 const TESTNET_DIR: &str = "testnet";
 
-/// Validators file name.
-const VALIDATORS_FILE_NAME: &str = "validators.txt";
-
 const VALIDATOR_KEYS: [&str; 3] = [
     "nHUSqn9qjEF7JJkVqvY7BFLMKdqP5KLLEjo5oB4QH43ADDndRawB",
     "nHUEsvSFTf1Snr7ZUdLxjcMW6PKcMrwwXCGZBg6xb1ePG8R4C3TS",
     "nHUuYdS49cPfRmCXPTwu7MVVFZFFmfG7y5sRttirVMhwuD7xStQp",
 ];
 
+pub fn get_validator_token(file_idx: usize) -> String {
+    match file_idx {
+        0 => include_str!("token0.txt").into(),
```

Comment:
I think I had a comment suggesting renaming those to `validator_tokenN.txt` :detective: